werkomgeving/gegeven/index.md

```diff
@@ -20,29 +20,29 @@ In bovenstaande voorbeelden wordt met [Jan] de verwijzing bedoeld naar een speci
 
 ## Een gegevenstype
 
-Een gegevenstype is *een typering van gelijksoortige gegevens*. Met een gegevenstype beschrijven we hoe een afzonderlijk gegeven er uit ziet, waarbij bovenstaande vier elementen terugkomen. Van een gegevenstype wil je de volgende zaken weten:
+Een gegevenstype is *een typering van gelijksoortige gegevens*. Met een gegevenstype beschrijven we hoe een afzonderlijk gegeven er uit ziet, waarbij bovenstaande vier elementen terugkomen. Van een gegevenstype wil je de volgende zaken weten: 
 
-1. Hoe refereer je naar het object waarover het gegeven gaat? Welke identificerende eigenschappen gebruik je daarbij?
```

Comment:
De formulering lijkt er op te duiden dat dit iets is dat je van een object wil weten. Ik kan me wel voorstellen dat je de formulering aanpast naar ‘_Is het gegeven identificerend voor het object waar het gegeven betrekking op heeft?_’.

Reply:
Eens dit is niet iets wat je van een gegevenstype wilt weten, maar juist van het object.

Let op: Het is ook mogelijk om gegevens uit te drukken over objecten waar, van het object zelf, geen identificerende gegevens (gemodelleerd) zijn.

Reply:
Van een gegeven wil je ook weten waar het over gaat. ALs je expliciet gegevensobjecten introduceert, als groep van gegevens met 1 hoofdonderwerp, dan is het voldoende om het hoofdonderwerp te duiden, omdat alle gegevens dan over dit hoofdonderwerp gaan. Maar als je gegevens afzonderlijk ziet (eigenlijk dus: gegevensobjecten die bestaan uit precies één gegeven), dan heb je wel zoiets nodig. Zoals in een normaal zinnetje, een feittype of een RDF triple. Alleen in relationele databases bij objectoriëntatie is het naders, maar da's wel het overgrote deel van de huidige databases :-D. In de tekst over gegevensobjecttypen is dit verder uitgewerkt.